actors/builtin/miner/policy.go

```diff
@@ -6,23 +6,19 @@ import (
 	power "github.com/filecoin-project/specs-actors/actors/builtin/power"
 )
 
-// An approximation to chain state finality.
+// An approximation to chain state finality (should include message propagation time as well).
 const ChainFinalityish = abi.ChainEpoch(500) // PARAM_FINISH
 
-// Lookback from current epoch from which to obtain a PoRep challenge.
-// TODO: HS why is this value unused?
-const PoRepLookback = ChainFinalityish // Should be approximately chain ~finality. PARAM_FINISH
```

Comment:
This is to be used in subsystem code for drawing the sealRandEpoch, not on actors.